### ISSUE TITLE
Threads memory is no longer remembered nor are (you) working

### ISSUE THREAD

**Issue by CorWilson** [ccd0/4chan-x#1397]
This happened a day ago. I figured it was just some kind of temporarily error but now it's still around.

1) Any thread I've been to before no longer saves it's place. Now when I arrive, it assumes I'm always in a new thread with the current number of replies.

2) I no longer get any notifications of someone quoting me. In fact, the site treats me like a completely different person sometimes, despite me being the same IP. I even have a case where I submit, but the extension doesn't realize I did posted so doesn't give me a countdown of the next submission.

**Comment by saxamaphone69** [comment]
Which version of Firefox? 4chan X? Greasemonkey?

Are there any errors in your console?

What other extensions are you using? Does it still occur if you disable all of them except for 4chan X/userscript manager? Does this happen on a fresh profile?

**Comment by ccd0** [comment]
Good chance it's #1323.

**Comment by CorWilson** [comment]
Firefox 47.0.2
4chan x 1.13.8.7
Greasemonkey 3.11

I disabled all extension but it but the problem still remains. In fact, I realize not even the native 4chan settings are working now in terms of scrolling to your recent thread position or saying if I posted or not. However I don't know whether the native extension had that functionality in the first place, so I can't be sure if that confirms anything.

**Comment by ccd0** [comment]
You might also want to check that you didn't disable those features in the settings.
Check "Remember Last Read Post" and "Remember Your Posts".

>not even the native 4chan settings are working now

Just to check, did you make sure to turn it on? You have to uncheck the "Disable the native extension" box and click the "Save Settings" button.

**Comment by ccd0** [comment]
If you indeed enabled the native extension and it still didn't work, very likely it's a Firefox profile problem. You should test in a fresh profile.
https://support.mozilla.org/en-US/kb/profile-manager-create-and-remove-firefox-profiles

**Comment by CorWilson** [comment]
Yes I have those two checked. Haven't touched any of the settings in a long time other than to edit my css. I also did uncheck the disable native extension button. I'll try with a fresh profile and see what happens.

**Comment by CorWilson** [comment]
Alright. I tried it in a new profile and the extension worked fine. Looks like I'll need to figure out what is specifically targeting it.
